### PR TITLE
feat: added customData field

### PR DESCRIPTION
**Pull request by alefcarlos**
it helps to store any data to retrieve later, for example documentId from Firestore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ ios/Runner/GeneratedPluginRegistrant.*
 
 lib/generated/
 res/
+
+.vscode/
```
Comment (Dn-a) [inline .gitignore]: add in .gitignore
unnecessary

File: example/lib/main.dart [hunk 2]
```diff
@@ -64,540 +81,536 @@ class _MyHomePageState extends State<MyHomePage> with SingleTickerProviderStateM
...
-            'first',
-            'return',
-            'lollipop',
-            'loop',
-        ]
-    );
+    _list.forEach((item) {
+      _selectableTags.add(Tag(
+          id: cnt,
+          customData: 'custom - ${cnt.toString()}',
```
Comment (alefcarlos) [inline example/lib/main.dart]: This one

File: lib/selectable_tags.dart [hunk 3]
```diff
@@ -408,12 +408,13 @@ class Tag
...
     bool active;
     int length;
+    dynamic customData;
 
 
     @override
     String toString()
     {
-        return '<TAG>\n id: $id;\n title: $title;\n active: $active;\n charsLength: $length\n<>' ;
+        return '<TAG>\n id: $id;\n customData: $customData;\n title: $title;\n active: $active;\n charsLength: $length\n<>' ;
```
Comment (alefcarlos) [inline lib/selectable_tags.dart]: And also here!

**Comment by Dn-a** [comment]
@alefcarlos another thing ... take the original source code of the main.dart file and enter the changes without adding indents. In the readme file add a short description.
thanks

**Comment by alefcarlos** [comment]
@Dn-a , It is done ! ;)

> @alefcarlos another thing ... take the original source code of the main.dart file and enter the changes without adding indents. In the readme file add a short description.
> thanks

**Pull request merged**